cmk/special_agents/agent_allnet_ip_sensoric.py

```diff
@@ -63,7 +63,7 @@ def get_allnet_ip_sensoric_info(host_address, opt_debug):
         raise RequestError('Error during http call')
 
     infos = handle.info()
```

Comment:
I guess we also need .decode() here, right? Could you please try this out by running the agent on the command line with the --debug option and checking the output?

Reply:
`handle.info()` returns a http.client.HTTPMessage object which is only used in the debug output `sys.stdout.write('Server: %s\n' % infos['SERVER'])`.

`infos['SERVER']` is already a string, so no decode() is needed here.

Reply:
Ok, thanks!